GNUmakefile

```diff
@@ -52,6 +52,7 @@ CSTOR_BASE_IMAGE= openebs/cstor-base:${BASE_TAG}
 # Specify the name for the binaries
 MAYACTL=mayactl
 APISERVER=maya-apiserver
+WEBHOOK=admission-webhook
```

Comment:
better to change the name to reflect openebs? cc: @AmitKumarDas

Reply:
and same applies for image name as well

Reply:
Pod name will be having openebs for example --> `openebs-admission-controller` or `openebs-admission-webhook `